LearningEngine/src/Graphics/Renderer/Model.cpp

```diff
@@ -9,10 +9,8 @@
 
 #include "API/API.h"
 
-Model::Model(std::filesystem::path path)
+Model::Model(std::filesystem::path path) : Model(path, Material(ShaderLibrary::GetShader("DefaultShader")))
 {
-	Material defMaterial = ShaderLibrary::GetShader("DefaultShader");
-	Model(path, defMaterial);
 }
 
```

Comment:
I think this was the original intention, since the old version just makes a new Model inside the constructor and discards it immediately.

I'm guessing this area was going to be refactored soon, but I figured it wouldn't hurt to change this anyway.